tools/nessus/nessus.md

```diff
@@ -137,9 +147,57 @@ Set-SmbServerConfiguration -AutoShareServer $False -AutoShareWorkstation $False
 
 ### Scanning Linux
 
-**Linux** does support credentialed scans via SSH key authentication.
+**Linux** does support credentialed scans via SSH key authentication.[^linux-scan] The account used for scanning must have root privileges.
+
+*Notice: If the shell variable PS1 is shorter than 4 letters, the scan time may drastically increase.*[^ps1]
+
+#### Generate SSH public and private key pair
+
+It does not matter where and how the key pair is created. However, it is important that the defined Tenable Nessus user owns the keys.
+
+Create a key pair with `ssh-keygen -t ed25519`. It is recommended to set **nessus** as the filename. Then, choose a passphrase (or choose an empty password by pressing `Return` twice). If you specify a passphrase, you must specify it in **Policies** > **Credentials** > **SSH settings** for Tenable Nessus to use key-based authentication.
+
+The private key will **stay on the server running Nessus**. The public key will be distributed to all clients that are scanned.
+
+To add a user for Nessus to login with, use the `adduser <username>` command. A prompt asking for the password and some optional information will appear. It does not matter what is entered in the prompt.
+
+Then, add the user to the `sudo` group with `usermod -aG sudo <username>`. Commands issued with sudo by this user will now run with root privileges.[^sudo-user]
+
+You must create the directory under the new account’s home directory to hold the public key. By default, the key file is copied under `/home/<username>/.ssh` and added to the `authorized_keys` file.
+
+```sh
+chown -R <username>:<username> /home/<username>/.ssh/
+chmod 700 /home/<username>/.ssh
+chmod 600 /home/<username>/.ssh/authorized_keys
+```
+
+Tenable Nessus encrypts all passwords stored in policies. However, Tenable recommends using SSH keys for authentication rather than SSH passwords. This helps ensure that someone does not use the same username and password you are using to audit your known SSH servers to attempt a login into a system that may not be under your control.
+
+When no scan is running, the user account can be disabled to forbid any login (even SSH) by `sudo usermod --expiredate 1 <username>`. Reenable the account with `sudo usermod --expiredate "" <username>`.[^disable-account-linux]
+
+To verify that the login over SSH works, use the `id` command:
+
+```sh
+ssh -i <ssh_private_key> nessus-scan@<scanned_host> id
+```
+
```

Comment:
known_hosts file?
https://tenable.my.site.com/s/article/How-to-create-known-hosts-file-for-Nessus-scans-using-openssh-clients?language=en_US
https://de.linux-console.net/?p=16101

ssh-keyscan <scanned_host> >> ~/.ssh/known_hosts
execute on the host running nessus

file saved in /opt/nessus/var/nessus/user/\<user\>/files